how-to/customize-workspace/client/src/modules/auth/example/auth-provider.ts

```diff
@@ -0,0 +1,277 @@
+let logInfo = console.log;
+let logWarning = console.warn;
+let logError = console.error;
+let authRequiredCallback: (authenticationRequired: boolean) => void;
+let authenticated: boolean;
+let authOptions: ExampleOptions;
+interface ExampleOptions {
```

Comment:
Move this into shapes.ts and rename to ExampleAuthOptions or similar

Reply:
This is specific to the sample auth module so should really be in the higher levels as in theory this module could be in common or somewhere else.